gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/DiskStoreImpl.java

```diff
@@ -268,6 +273,11 @@
   // /** delay for slowing down recovery, for testing purposes only */
   // public static volatile int recoverDelay = 0;
 
+  // extra reserved space to be freed when standby oplog is in use,
+  // so that renaming of standby oplog, & creation of krf idxkrf etc is successful
+  // This is to be specified in MB & default is 1 GB , i.e 1024 MB
```

Comment:
@ahshahid do you mean this has to be specified in KB instead of MB
because in preblow you are multiplying it by 1024 * 1024
So total will be 1024 * 1024 * 1024.
So I guess just comment error?

Reply:
@kneeraj , I am not sure where the mistake is... 
The comment says:
The value needs to be specified in MB. The default is 1GB ( which in MB specification is 1024 MB).
So 1024 MB in bytes for preblowing would be  ( mb to kb to bytes)
1024 * 1024 * 1024.
right?